__tests__/app.test.js

```diff
@@ -28,4 +28,27 @@ describe("GET /api/topics", () => {
         });
       });
   });
+  test("status 404: not found when passed an invalid endpoint", () => {
+    return request(app).get("/api/apples").expect(404);
+  });
+});
+describe("GET /api/articles/:article_id", () => {
+  test("status 200: responds with an Article object", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body;
+        console.log(article, body);
+        expect(article.to.be.an("object"));
+        expect(article).toMatchObject({
+          title: expect.any(String),
```

Comment:
I think that, because in this case we know exactly what object will be coming back (because we are testing for a request to article 1) - we can test for the exact information that we are expecting, using the data in our database as an example. So we could do `expect(article).toEqual({title: "whatever the title is for article 1 in the database"})` etc.